app/controllers/devise_token_auth/application_controller.rb

```diff
@@ -37,7 +37,8 @@ def is_json_api
       return false unless defined?(ActiveModel::Serializer)
       return ActiveModel::Serializer.setup do |config|
         config.adapter == :json_api
-      end
+      end if ActiveModel::Serializer.respond_to?(:setup)
+      return ActiveModelSerializers.config.adapter == :json_api
```

Comment:
I didn't realize that https://github.com/lynndylanhurley/devise_token_auth/pull/648 was targeted at AMS 0.9.x. It's been a while since I used it and I didn't see any mention of version in the pr.